conda_package_builder/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM centos:7
+
+ARG PYTHON_VERSION=3.7m
+ARG CONDA_VERSION=4.5.4
+
+ENV OSPL_HOME=/opt/OpenSpliceDDS/V6.9.0/HDE/x86_64.linux
+
+ENV PYTHON_BUILD_VERSION=${PYTHON_VERSION}
+ENV PYTHON_BUILD_LOCATION=/home/saluser/miniconda3
+
+USER root
+
+RUN yum install -y mlocate
+RUN adduser -u 1004 -m -s /bin/bash saluser
+RUN yum install -y iproute gcc-c++ make ncurses-libs xterm xorg-x11-fonts-misc java-1.7.0-openjdk-devel maven python-devel swig git tk-devel wget bzip2
+
+COPY lsst-ts-nexus.repo /tmp/lsst-ts-nexus.repo
+
+RUN cat /tmp/lsst-ts-nexus.repo >> /etc/yum.conf && yum install -y OpenSpliceDDS-6.9.0
+USER saluser
+WORKDIR /home/saluser
+
+RUN wget --no-verbose https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh && chmod +x miniconda.sh && ./miniconda.sh -b
```

Comment:
So, if we are setting the `CONDA_VERSION` as an argument, I think we might as well pull the same version instead of latest.

Reply:
I think pulling latest would be beneficial as it allows us to catch any potential problems early.

Reply:
Changing wget to curl -O will save you yum install wget. curl is preferred, as it's available at base, and build on top of C library